Makefile.work

```diff
@@ -118,7 +118,8 @@ SONIC_BUILD_INSTRUCTION :=  make \
                            KERNEL_PROCURE_METHOD=$(KERNEL_PROCURE_METHOD) \
                            HTTP_PROXY=$(http_proxy) \
                            HTTPS_PROXY=$(https_proxy) \
-                           SONIC_ENABLE_SYSTEM_TELEMETRY=$(ENABLE_SYSTEM_TELEMETRY)
+                           SONIC_ENABLE_SYSTEM_TELEMETRY=$(ENABLE_SYSTEM_TELEMETRY) \
+                           $(SONIC_OVERRIDE_BUILD_VARS)
```

Comment:
can you make this as a run time option. it is better not to build another image at all.

Reply:
We can not have one image for simx and real hw, because of the hw-management patch.
Can this delayed install feature help with that? Is there some wiki on that feature ? 
Maybe we could build both patched and unpatched hw-mgmt and install on system accordingly?

Reply:
check this 
```
platform/broadcom/one-image.mk:$(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(DELL_S6000_PLATFORM_MODULE) \
platform/broadcom/platform-modules-s6000.mk:$(DELL_S6000_PLATFORM_MODULE)_PLATFORM = x86_64-dell_s6000_s1220-r0
```

you can create two hw-mgmt package, one for simx and one for real hw, and then based on the platform onie string, you can selectively to install either one of it.